include/mbedtls/config.h

```diff
@@ -340,8 +335,6 @@
 //#define MBEDTLS_ECP_ALT
 
 /**
- * \def MBEDTLS_MD2_PROCESS_ALT
- *
  * MBEDTLS__FUNCTION_NAME__ALT: Uncomment a macro to let mbed TLS use you
```

Comment:
A `\def` line should stay at the top of the comment here so that it won't attach to the wrong definition when processing this file with all the `def MBEDTLS_xxx_PROCESS_ALT` commented out.
```suggestion
 * \def MBEDTLS_SHA256_PROCESS_ALT
 *
 * MBEDTLS__FUNCTION_NAME__ALT: Uncomment a macro to let mbed TLS use you
```